ramblings/holey_domains.jl

```diff
@@ -0,0 +1,111 @@
+# Generating dummy column names & data
```

Comment:
This code (or some evolution of it) could be handy if we ever want automated tests for the backend